server/form-pages/apply/risk-and-need-factors/access-and-healthcare/accessNeeds.ts

```diff
@@ -66,7 +71,13 @@ export default class AccessNeeds implements TasklistPage {
   }
 
   next() {
-    if (this.body.additionalNeeds.includes('mobility')) return 'access-needs-mobility'
+    if (
+      (this.body.additionalNeeds as Array<AdditionalNeed>).some(need => {
+        return ['mobility', 'pregnancy', 'visualImpairment'].includes(need)
+      })
```

Comment:
Might be worth extracting this out into a function called something like `furtherQuestionsNeeded`?